git-extra/git-credential-chooser.c

```diff
@@ -9,6 +9,148 @@ static size_t helper_nr, selected_helper;
 #define ID_ABORT   IDCANCEL
 #define ID_USER    2000
 
+static LPWSTR parse_script_interpreter(LPWSTR path)
+{
+#define MAX_SHEBANG 100
+	static WCHAR wbuf[2 * MAX_SHEBANG + 16];
+	char buf[MAX_SHEBANG + 5];
+	DWORD len = 0, eol = 0, count;
+	HANDLE h;
+
+	/* Skip detection for .exe, .bat, and .cmd */
+	count = wcslen(path);
+	if (count > 4 && path[count - 4] == L'.' &&
+	    (!wcscmp(path + count - 3, L"exe") ||
+	     !wcscmp(path + count - 3, L"bat") ||
+	     !wcscmp(path + count - 3, L"cmd")))
+		return NULL;
+
+	h = CreateFileW(path, GENERIC_READ, FILE_SHARE_READ, NULL,
+			       OPEN_EXISTING, FILE_ATTRIBUTE_NORMAL, NULL);
```

Comment:
Wrong indent? Or Github using 8-spaced TABs and your editor 4-spaced?

Reply:
Sharp eyes! I fixed it.